x/auth/spec/05_vesting.md

```diff
@@ -22,20 +22,18 @@
 
 ## Intro and Requirements
 
-This specification describes the vesting account implementation for the Cosmos Hub.
-The requirements for this vesting account is that it should be initialized
-during genesis with a starting balance `X` and a vesting end time `T`.
+This specification defines the vesting account implementation that is used by the Cosmos Hub. The requirements for this vesting account is that it should be initialized during genesis with a starting balance `X` and a vesting end time `T`. A vesting account may be initialized with a vesting start time `T'` and a number of vesting periods `P`. If a vesting start time is included, the vesting period will not begin until start time is reached. If vesting periods are included, the vesting will occur over the specified number of periods.
```

Comment:
Markdown should be wrapped at 80-120 chars. @marbar3778 is there a linter for this?

Reply:
yeah, worst case you can define break lines on the IDE's settings